azurerm/resource_arm_batch_pool_test.go

```diff
@@ -394,7 +463,7 @@ EOF
     offer     = "UbuntuServer"
     sku       = "16.04.0-LTS"
     version   = "latest"
-  }
+	}
```

Comment:
Could we fi the indentation here?
```suggestion
  }
```